pandas/tests/arrays/test_integer.py

```diff
@@ -603,6 +603,15 @@ def test_groupby_mean_included():
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_mean_no_overflow():
```

Comment:
Can you put this in pandas/tests/groupby/test_function.py instead?

Reply:
Done.